protocols/autonat/src/v2/server.rs

```diff
@@ -0,0 +1,5 @@
+mod behaviour;
+mod handler;
+
+pub use behaviour::Behaviour;
+pub use behaviour::StatusUpdate;
```

Comment:
Let's call this `Event` like in other protocols.

Reply:
I renamed it